src/data/en/index.ts

```diff
@@ -943,3 +999,78 @@ export const loggedInData = {
     }
   },
 }
+
+export const kratosMailStrings = {
+  recovery: {
+    valid: {
+      subject: '👉 Access to your Serlo account',
+      'body.plaintext': `👋 Hi {{ .Identity.traits.username }},
+are you trying to get access to your account at serlo.org? If not please just ignore this mail.
```

Comment:
```suggestion
Are you trying to get access to your account at serlo.org? If not please just ignore this mail.
```
In English you should capitalize a word after greetings.

Reply:
👍